Tests/Marketplace/install_packs.sh

```diff
@@ -17,6 +17,4 @@ echo "Auth loaded successfully."
 
 echo "starting configure_and_install_packs ..."
 
-python3 ./Tests/Marketplace/configure_and_install_packs.py -s "$SECRET_CONF_PATH" --ami_env "$1" --branch "$CI_COMMIT_BRANCH" --build_number "$CI_PIPELINE_ID" --service_account $GCS_MARKET_KEY -e "$EXTRACT_FOLDER" --cloud_machine "$CLOUD_CHOSEN_MACHINE_ID" --cloud_servers_path $XSIAM_SERVERS_PATH --pack_ids_to_install "$ARTIFACTS_FOLDER/content_packs_to_install.txt" --cloud_servers_api_keys $XSIAM_API_KEYS
-
-exit $RETVAL
```

Comment:
Why we do not want to exit on failure if the pack's installation failed? 
(It is the original design in XSIAM, and in XSOAR if pack installation fails it catches and not failing the build).